packages/components/socialFeed/Map/Map.types.ts

```diff
@@ -0,0 +1,10 @@
+import { StyleProp, ViewStyle } from "react-native";
+
+import { CustomLatLngExpression, PostCategory } from "@/utils/types/feed";
+
+export interface MapProps {
+  creatingPostLocation?: CustomLatLngExpression; // When the user is adding a location to a post he's creating
+  creatingPostCategory?: PostCategory; // When the user is adding a location to a post he's creating
```

Comment:
```suggestion
  creatingPostCategory?: PostCategory; // When the user is adding a category to a post he's creating
```

Reply:
That's **location** yes ^^
The `Map` is shown at two places:
- When the user consults posts on Map Feed
- When the user makes a post and add a location to it using the `MapMpdal`

On `MapModal`, we show a post icon on the map when the user has selected an address.
To do that, `creatingPostLocation` and `creatingPostCategory` can be specified
![image](https://github.com/user-attachments/assets/4a628a4f-0ddb-4cf7-8b8f-48fc49e44d1b)